scripts/generate-policy-config.py

```diff
@@ -30,6 +30,7 @@
         'osd-delete-backplane-script-resources',
         'osd-delete-backplane-serviceaccounts',
         'osd-backplane-managed-scripts',
+        'osd-logging',
```

Comment:
The root directory only for extended logging support.  We will never bring this into HCP.
What needs to be included is `deploy/osd-logging/unsupported`